src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbol.cs

```diff
@@ -1285,7 +1285,8 @@ private ImmutableArray<ParameterSymbol> ComputeParameters(Binder binder, BasePro
                 }
                 else if ((object)_setMethod != null && param.Name == ParameterSymbol.ValueParameterName)
                 {
-                    diagnostics.Add(ErrorCode.ERR_DuplicateGeneratedName, _location, param.Name);
+                    // note: there can only ever be one location for param, as it cannot be used in partial for a property
+                    diagnostics.Add(ErrorCode.ERR_DuplicateGeneratedName, param.Locations[0], param.Name);
```

Comment:
If `param` does not have a location, what is the odds that `_location` would not be null? `FirstOrDefault` alone should be fine?